doc/run_experiment.rst

```diff
@@ -506,8 +506,18 @@ fixed_parameters *(Optional)*
 """""""""""""""""""""""""""""
 
 List of dicts containing parameters you want to have fixed for each
-classifier in :ref:`learners` list. Any empty ones will be ignored
-(and the defaults will be used).
+learner in :ref:`learners` list. Any empty ones will be ignored
+(and the defaults will be used). If there is a conflict between parameters
+specified in :ref:`param_grids` and :ref:`fixed_parameters`, an exception
+will be raised (unless the conflicting values are the same). Furthermore,
+since default parameter grids can be used either by not specifying
+parameter grids at all or by specifying blank lists for specific
+parameter grids corresponding to different learners (i.e., in cases where
+you want to use default parameter grids for some learners, but for others
+you want to specify the parameter grids yourself), there is a potential for
+conflicts between fixed parameters and default parameter grid values that
+are not obvious initially. When such conflicts arise, the fixed parameter
+values will take precedence and a warning will be logged.
```

Comment:
These feel like very similar situations (there is a conflict between `fixed_parameters` and `param_grids`. In one case an exception is raised, but in the other a warning is issued. This feels a little strange though I see how the cases are subtly different.